agent/api/task/task.go

```diff
@@ -2120,7 +2106,7 @@ func (task *Task) shouldOverridePIDMode(container *apicontainer.Container, docke
 				field.TaskID: task.GetID(),
 			})
 			task.SetDesiredStatus(apitaskstatus.TaskStopped)
-			return false, ""
+			break
```

Comment:
nit - can probably just return here, same with the other case blocks.

Reply:
fair